src/pyagxrobots/pysdkugv.py

```diff
@@ -0,0 +1,236 @@
+        self.basetype='scout'
+        self.motronum= 4
+        self.scoutbase=agxrobots.UGV()
+        self.baseversion=self.scoutbase.base_version
+        self.max_linear=1.5
+        self.max_angular=0.5235
+        self.max_lateral=0.0
+        self.min_linear=-self.max_linear
+        self.min_angular=-self.max_angular
+        self.min_lateral=-self.max_lateral
+        if self.baseversion==1:
+            self.ScoutActuator1=ActuatorStateMessageV1(motro_id=1)
+            self.ScoutActuator2=ActuatorStateMessageV1(motro_id=2)
+            self.ScoutActuator3=ActuatorStateMessageV1(motro_id=3)
+            self.ScoutActuator4=ActuatorStateMessageV2(motro_id=4)
+        elif self.baseversion==2:
+            self.ScoutActuator1=ActuatorStateMessageV2(motro_id=1)
+            self.ScoutActuator2=ActuatorStateMessageV2(motro_id=2)
+            self.ScoutActuator3=ActuatorStateMessageV2(motro_id=3)
+            self.ScoutActuator4=ActuatorStateMessageV2(motro_id=4)        
+
+    def  SetMotionCommand(self,linear_vel=0.0,angular_vel=0.0):
+        if self.baseversion==2:
+            if linear_vel>self.max_linear:
+                linear_vel=self.max_linear
+            elif linear_vel<self.min_linear:
+                linear_vel=self.min_linear
+            if angular_vel>self.max_angular:
+                angular_vel=self.max_angular
+            elif angular_vel<self.min_angular:
+                angular_vel=self.min_angular
+        self.scoutbase.SetMotionCommand(linear_vel,angular_vel,0,0)
+    def SetLightCommand(self,f_mode,f_value,r_mode,r_value):
+        self.scoutbase.SetLightCommand(f_mode,f_value,r_mode,r_value)
+        # self.version= 
+
+    def EnableCAN(self):
+        if self.baseversion==2:
+            self.scoutbase.EnableCANCtrl()   
+        
+
+class ScoutMiniBase(GetRobotStae):
+    def __init__(self):
+        self.basetype='scoutMini'
+        self.motronum= 4
+        self.scoutminibase=agxrobots.UGV()
+        self.baseversion=self.scoutminibase.base_version
+        self.max_linear=3.0
+        self.max_angular=2.5235
+        self.max_lateral=2.0
+        self.min_linear=-self.max_linear
+        self.min_angular=-self.max_angular
+        self.min_lateral=-self.max_lateral
+        if self.baseversion==1:
+            self.ScoutMiniActuator1=ActuatorStateMessageV1(motro_id=1)
+            self.ScoutMiniActuator2=ActuatorStateMessageV1(motro_id=2)
+            self.ScoutMiniActuator3=ActuatorStateMessageV1(motro_id=3)
+            self.ScoutMiniActuator4=ActuatorStateMessageV2(motro_id=4)
+        elif self.baseversion==2:
+            self.ScoutMiniActuator1=ActuatorStateMessageV2(motro_id=1)
+            self.ScoutMiniActuator2=ActuatorStateMessageV2(motro_id=2)
+            self.ScoutMiniActuator3=ActuatorStateMessageV2(motro_id=3)
+            self.ScoutMiniActuator4=ActuatorStateMessageV2(motro_id=4)
+
+    
+    def  SetMotionCommand(self,linear_vel=0.0,angular_vel=0.0,lateral_velocity=0.0):
```

Comment:
Hi, what is the correct way to handle angular velocity alongside with the linear velocity?
It seems like the robot is skidding on my code, although while trying to use the more control I get a smooth turning